docs/content/docs/tutorials/aurora-serverless-v2.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "Manage an Aurora Serverless v2 cluster with the ACK RDS Controller"
+description: "Create an Aurora Serverless v2 cluster from an Amazon Elastic Kubernetes Service (EKS) deployment."
+lead: "Create and manage an Aurora Serverless v2 cluster directly from Kubernetes"
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 45
+toc: true
+---
+
+In this tutorial you will learn how to create and manage
+[Aurora Serverless v2](https://aws.amazon.com/rds/aurora/serverless/) instances
+from an Amazon Elastic Kubernetes (EKS) deployment.
```

Comment:
I think it would be useful to have a paragraph here explaining the difference between Aurora Serverless v2 and original Aurora Serverless. Don't assume the reader knows this (I didn't...) :)